en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,146 @@
+# File Connector Example
+
+The File Connector allows you to connect to different file systems and perform various operations. The File Connector uses the Apache Commons VFS I/O functionalities to execute operations.
+
+WSO2 EI File Connector introduces the independent operations related to the file system and allows you to easily manipulate files based on your requirement. The file streaming functionality using Apache Commons I/O lets you copy large files and reduces the file transfer time between two file systems resulting in a significant improvement in performance that can be utilized in file operations.
+
+File Connector can be used to perform operations in the local file system as well as in a remote server such as FTP and SFTP. 
+
+## What you'll build
+
+This example explains how to use File Connector to create a file in the local file system and read the particular file. The user sends a payload specifying which content is to be written to the file. Based on that content, a file is created in the specified location. Then the content of the file can be read as an HTTP response by invoking the other API resource upon the existence of the file. 
+
+It will have two HTTP API resources, which are `create` and `read`. 
+
+* `/create `: It will create a file with the content that the user specifies in the payload. 
+<p><img src="../../../assets/img/connectors/FileConnector-03.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/></p>
+
+* `/read `: It will first check if the file exists. If so it will read the content of the file. 
+<p><img src="../../../assets/img/connectors/FileConnector-02.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/></p>
+
+## Configure the connector in WSO2 Integration Studio
+
+1. Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
+[Configure the Connector and the Connector Exporter Project](../importing-connector-to-integration-studio.md).
```

Comment:
We need to verify how this behaves at MKDocs. 
@samgnaniah is this the correct way of linking to the common page?

Reply:
No, this should be in the following format:

"<!-- INCLUDE_MD: ../../<file-name>.md -->"

Reply:
Better to test it out locally using mkdocs and see.